kolibri/core/auth/test/test_api.py

```diff
@@ -1170,6 +1173,91 @@ def test_anonymous_no_retrieve_user(self):
         self.assertEqual(response.status_code, 404)
 
 
+class FacilityUserOrderingTestCase(APITestCase):
+    @classmethod
+    def setUpTestData(cls):
+        provision_device()
+        cls.facility = FacilityFactory.create()
+        cls.superuser = create_superuser(cls.facility)
+        cls.facility.add_admin(cls.superuser)
+        cls.user1 = FacilityUserFactory.create(facility=cls.facility, username="mario")
```

Comment:
I am a little surprised that the date_joined sort works on Windows here. Just for safety, it might be good to explicitly edit the date_joined field to give specific dates that are separated in time.